frontend/src/features/Train/features/Tabular/components/TabularReviewStep.tsx

```diff
@@ -23,7 +23,7 @@ const TabularReviewStep = ({
   );
   const {
     handleSubmit,
-    formState: { errors, isDirty },
```

Comment:
@dwu359, is the `isDirty` variable planned to be in use here. eslint complained that `isDirty` is unused var, so we assumed no use case for this variable. Correct me if I'm wrong?

Reply:
Yeah I don't think that variable is being used, it says that on my end too

Reply:
so can we remove it @dwu359?